modules/monitoring/application/forms/Command/Object/AddCommentCommandForm.php

```diff
@@ -81,6 +129,9 @@ public function onSuccess()
             if (($persistent = $this->getElement('persistent')) !== null) {
                 $comment->setPersistent($persistent->isChecked());
             }
+            if ($this->getElement('expire')->isChecked()) {
+                $comment->setExpireTime($this->getElement('expire_time')->getValue()->getTimestamp());
+            }
```

Comment:
Please add an empty line before and after this block. We require an empty line after closing braces.